### PR TITLE
added nuspec and pipeline yml for Continous Integration

### PR DESCRIPTION
**Pull request by winstongubantes**

### CODE REVIEW

File: build/Plugin.nuspec [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	   <file src="..\src\JsonRpcSharp.UnixIpcClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\netstandard1.3" />
+	   <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\netstandard1.3\JsonRpcSharp.*" target="lib\netstandard1.3" />
+	   
+	   <!--netstandard2.0-->
+       <file src="..\src\JsonRpcSharp.TcpClient\bin\Release\netstandard2.0\JsonRpcSharp.*" target="lib\netstandard2.0" />
+	   
+       <!--Net Core 2.1-->
+       <file src="..\src\JsonRpcSharp.IpcClient\bin\Release\netcoreapp2.1\JsonRpcSharp.*" target="lib\netcoreapp2.1" />
+	   <file src="..\src\JsonRpcSharp.WebSocketClient\bin\Release\netcoreapp2.1\JsonRpcSharp.*" target="lib\netcoreapp2.1" />
+   </files>
```
Comment (knocte) [inline build/Plugin.nuspec]: hey @winstongubantes there is a mix of tabs and spaces in this file, can you fix please?

Comment (knocte) [reply build/Plugin.nuspec]: also can you rename the file from `Plugin.nuspec` to `JsonRpcSharp.nuspec`? thanks

Comment (winstongubantes) [reply build/Plugin.nuspec]: already updated the file

**Comment by knocte** [comment]
Thanks Winston, only nitpick is that you don't have your git configuration properly set, look: https://github.com/nblockchain/JsonRpcSharp/commit/b627e9e8566a8f82a18672cf6f3a747d88553469.patch -> it says "Winston <Winston@DESKTOP-M61MF74>".

The way to fix is this: in your terminal type this:

```
git config --global user.name "Winston Gubantes"
git config --global user.email winston.gubantes@gmail.com
```

After that, to re-commit, you could do:

```
git reset HEAD~8
```

(Because you did 8 commits.) And then commit again (with just one commit).

**Comment by knocte** [comment]
(Oh, and after that last commit, you need to push , with force, with the  `-f` flag.)

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    msbuildArguments: '/t:restore;build /p:Configuration=Release'
+
+
+- task: NuGetCommand@2
+  displayName: 'NuGet pack'
+  inputs:
+    command: pack
+
+    packagesToPack: build/Plugin.nuspec
```
Comment (knocte) [inline azure-pipelines.yml]: oh I guess you need to change this because we renamed the file

**Comment by knocte** [comment]
The yml file still says "Plugin.nuspec" instead of "JsonRpcSharp.nuspec" :)

**Comment by knocte** [comment]
Hey @winstongubantes I think that now you have done a `reset` with a too high number, because if you click on the tab "Files changed" of the PR, you will see there are changes in the .fs files (probably undoing some previous commit), which we don't want.
Maybe it's faster to fix this by proposing a new PR :) Next time, we could jump into a sharing-screen session so I can show you a few git tips :)

**Comment by knocte** [comment]
Superseded by https://github.com/nblockchain/JsonRpcSharp/pull/6

**Pull request closed without merging**